src/Quasar/Advanced/Types.purs

```diff
@@ -14,33 +14,17 @@ import Data.Path.Pathy as Pt
 import Data.String as Str
 import Data.Traversable (traverse)
 
-import OIDCCryptUtils.JSONWebKey (JSONWebKey)
-import OIDCCryptUtils.Types (Issuer(..), ClientID(..), runClientID, runIssuer)
+import OIDC.Crypt.JSONWebKey (JSONWebKey)
+import OIDC.Crypt.Types (Issuer(..), ClientID(..), runClientID, runIssuer)
 
 data Operation
   = Add
   | Read
   | Modify
   | Delete
 
-
-instance eqOperation ∷ Eq Operation where
-  eq Read Read = true
-  eq Modify Modify = true
-  eq Delete Delete = true
-  eq _ _ = false
```

Comment:
I assume `eq Add Add` wasn't actually supposed to equal `false` here, and was just an oversight?